fatrace.c

```diff
@@ -308,6 +316,15 @@ print_event (const struct fanotify_event_metadata *data,
     } else if (option_timestamp == 2) {
         printf ("%li.%06li ", event_time->tv_sec, event_time->tv_usec);
     }
+
+    /* print user and group */
+    if (option_user == 1) {
+        if (got_procname)
+            printf("%i:%i ", ug.st_uid, ug.st_gid);
```

Comment:
Wrong condition -- this should check if `ug` is valid, not if the procname is valid. Also, the IDs have type `uint32_t`, so don't print them as `%i`, but `%u`.

Reply:
Fixed.